homeassistant/components/opple/manifest.json

```diff
@@ -5,5 +5,5 @@
   "documentation": "https://www.home-assistant.io/integrations/opple",
   "iot_class": "local_polling",
   "loggers": ["pyoppleio"],
-  "requirements": ["pyoppleio==1.0.5"]
+  "requirements": ["pyoppleio-legacy==1.0.8"]
```

Comment:
Side note: The license specified in `setup.py` (MIT) doesn't match the included license file (Apache 2.0).

https://github.com/tinysnake/python-oppleio-legacy/blob/f510d2327e68bf536c5e7b5ca61074d3413271f2/setup.py#L14

Reply:
> Side note: The license specified in `setup.py` (MIT) doesn't match the included license file (Apache 2.0).
> 
> https://github.com/tinysnake/python-oppleio-legacy/blob/f510d2327e68bf536c5e7b5ca61074d3413271f2/setup.py#L14
> 
> Choose one of them.

I just updated the license of the repo from Apache 2.0 to MIT, is it ok? I'm not quite sure what 'included license file' exactly means, sorry.

Reply:
I mean this file:

https://github.com/tinysnake/python-oppleio-legacy/blob/main/LICENSE

Good! Now the file and the config in `setup.py` match.